spec/factories/credential_factory.rb

```diff
@@ -3,23 +3,29 @@
     transient do
       login { "tester" }
       admin { false }
+      logged_in { true }
+      user_id { -1 }
       workflows { [] }
+      expired { false }
     end
 
     token { "fake" }
-    expires_at { 1.day.from_now }
     project_ids { workflows.map(&:project_id).uniq }
 
     after(:build) do |credential, evaluator|
-      credential.instance_variable_set(:@jwt_payload, {"login" => evaluator.login, "admin" => evaluator.admin})
+      credential.instance_variable_set(:@login, evaluator.login)
```

Comment:
These can go now, they aren't used anywhere so no need to set them up. I think we can remove the associated transient declarations too.

Reply:
yep actually that whole file is no longer necessary